python_modules/dagster/dagster/_core/definitions/source_asset.py

```diff
@@ -31,13 +32,13 @@ class SourceAsset(
     NamedTuple(
         "_SourceAsset",
         [
-            ("key", AssetKey),
+            ("key", PublicAttr[AssetKey]),
             ("metadata_entries", Sequence[Union[MetadataEntry, PartitionMetadataEntry]]),
```

Comment:
arguably, shouldn't this be public for people directly utilizing the `SourceAsset` object in some way?